### PR TITLE
APP-20450 design error code view

### PR DESCRIPTION
**Pull request by fellmann**

### CODE REVIEW

File: apps/swirl-docs/src/lib/docs/src/oasBuilder.ts [hunk 1]
```diff
@@ -185,9 +182,7 @@ export default class OASBuilder implements IOASBuilder {
               pathItemObject as HttpMethods
             );
 
-            const isInternal = operation.getExtension(
-              OASBuilder.X_FLIP_INTERNAL
-            );
+            const isInternal = FlipApiExtensions.getInternal(operation);
```
Comment (webspaceadam) [inline apps/swirl-docs/src/lib/docs/src/oasBuilder.ts:185]: very nice. Was on my personal To Do List already. Can cross it out now :D

File: apps/swirl-docs/src/components/Documentation/EndpointDescription.tsx [hunk 2]
```diff
@@ -121,6 +127,30 @@ export const EndpointDescription: FunctionComponent<EndpointDescription> = ({
             </div>
           </div>
         )}
+
+        {!!endpoint.globalErrorCodes?.length && (
```
Comment (webspaceadam) [inline apps/swirl-docs/src/components/Documentation/EndpointDescription.tsx:131]: thought (non-blocking): could be put in a component

**Review by webspaceadam — APPROVED** [review]
lgtm. just the one additional idea

File: apps/swirl-docs/src/components/Documentation/EndpointDescription.tsx (hunk 2 above)
Comment (fellmann) [reply apps/swirl-docs/src/components/Documentation/EndpointDescription.tsx:131]: I would prefer to split this large component in several sub-components outside this PR

Comment (webspaceadam) [reply apps/swirl-docs/src/components/Documentation/EndpointDescription.tsx:131]: sure. can we make a ticket for that?

Comment (fellmann) [reply apps/swirl-docs/src/components/Documentation/EndpointDescription.tsx:131]: APP-20517

**Pull request merged**